integtests/structs/instance.go

```diff
@@ -0,0 +1,72 @@
+package structs
+
+const (
+	InProgressOperationSate        OperationState       = "in progress"
+	InProgressOperationDescription OperationDescription = "creating service instance..."
+	SucceedOperationState          OperationState       = "succeeded"
+	SucceedOperationDescription    OperationDescription = "successfully created service instance"
+	FailedOperationState           OperationState       = "failed"
+	FailedOperationDescription     OperationDescription = "failed to create service instance"
+)
```

Comment:
1. A common style is declaring custom type right above constants that use it. And to group constants by type. Besides no need to declare type along with each const.
```go
type OperationState string

const (
	InProgressOperationSate OperationState = "in progress"
	SucceedOperationState                  = "succeeded"
	FailedOperationState                   = "failed"
)
...
```

2. I don't any usage of those constants in code. Use them!)